.bazelci/presubmit.yml

```diff
@@ -17,6 +17,20 @@ tasks:
     test_targets:
     - "//..."
 
+  cpan_compile_ubuntu2204:
+    name: CPAN Compile
+    platform: ubuntu2204
+    run_targets:
+    - "//perl/cpan/3rdparty:compiler"
+    - "//perl/cpan/3rdparty:compiler"
```

Comment:
You mention somewhere that the lockfile will be generated empty on the first run, is this why you have this twice? (same below)
If yes I think it would be nicer to fix that in the compiler implementation so users get the expected result on the first try

Reply:
The compilation will work first try, but because the rule takes a label to the lockfile, the lockfile must be made (by hand) before the full target can be built. This is run twice to show the results are usable since the compiler is consuming the dependencies it outputs with this.

Reply:
Sounds good, maybe a comment would be good to make that clear to future readers

Reply:
Updated!